x-pack/plugins/security_solution/cypress/e2e/data_sources/sourcerer.cy.ts

```diff
@@ -211,7 +203,7 @@ describe('Timeline scope', () => {
     });
     beforeEach(() => {
       visit(TIMELINES_URL);
-      waitForAlertsIndexToExist();
+      refreshUntilAlertsIndexExists();
```

Comment:
the alerts index always exists but this function also includes a page refresh which some tests seem to need

Reply:
Still catching up a bit here since leave - is this a new change? Previously I thought the alerts index only gets created on first write?

Reply:
Ignore my comment - just read through the comments you'd added!

Reply:
Yes! Sorry for being unclear, the alerts indices in the default space will be installed by the alerting plugin on Kibana start by default after this PR